packages/runtime/container-runtime/src/opLifecycle/opSplitter.ts

```diff
@@ -212,27 +212,31 @@ export const splitOp = (op: BatchMessage, chunkSizeInBytes: number): IChunkedOp[
 	);
 
 	const contentLength = op.contents.length;
-	const chunkN = Math.floor((contentLength - 1) / chunkSizeInBytes) + 1;
+	const chunkCount = Math.floor((contentLength - 1) / chunkSizeInBytes) + 2;
```

Comment:
This does not look like ideal. I.e. if there is a 10Mb batch with just 1-2 ops, then this will result in sending one pretty tiny chunk at the end. It's probably fine - the worst case here is we have a lot of batches like that, i.e. 10Mb+ of uncompressed content per batch of 1-2 ops, and a lot of batches like that - that's when we double number of ops with this proposal (and run the risk of making stuff worse RE op count throttling). Given this scenario is very unlikely and eventually will be fixed by Kian's work, I think I'm fine with it, but raising this comment to think if there are better ways to solve it.